post-start.d/60-replace-provd-py2-plugins.py

```diff
@@ -28,6 +28,7 @@
 }
 DEFAULT_REPO = 'https://provd.wazo.community/plugins/2/stable/'
 REPO_REGEX = re.compile(r'^(https?)://provd.wazo.community/plugins/1/(testing|stable|archive)/?$')
+NEW_REPO_REGEX = re.compile(r'^(https?)://provd.wazo.community/plugins/2/(testing|stable|archive)/?$')
```

Comment:
This definitely works, but you could also use the same regex and just add a group.
```python
REPO_REGEX = re.compile(r'^(https?)://provd.wazo.community/plugins/([12])/(testing|stable|archive)/?$')

match = REPO_REGEX.match(current_url)
if match:
    proto, version, variant = match.groups()
    
    if version != '1':
        return
```

Reply:
It's harder to read, no?

Reply:
I don't think so, but I guess it depends on your feelings on regular expressions. haha. Just a suggestion.

Reply:
I'd say they're equal, but personally I find the latter clearer, but I don't mind if you prefer the former. It was just a suggestion.